app/controllers/admin_controller.rb

```diff
@@ -39,6 +42,14 @@ def users
     # Filter by search text
     users = filter(users, search)
 
+    # Filter by status text
+    case status
+    when 'locked'
+      users = users.where.not(locked_at: nil)
+    when 'unlocked'
+      users = users.where(locked_at: nil)
+    end
```

Comment:
If users was nil for some reason, we should return an error here and not continue to the sort.

Reply:
out of curiosity, what would cause `users` to be `nil` at this point in the code? wouldn't it only either be the original value (line 38) or whatever the result of `filter` is (which can only be another valid AR query)?

Reply:
I also have this question because the worst it would be here is an empty array